includes/admin/class-wc-payments-admin.php

```diff
@@ -265,6 +265,14 @@ public function add_payments_menu() {
 					'path'   => '/payments/disputes/challenge',
 				]
 			);
+			wc_admin_register_page(
+				[
+					'id'     => 'wc-payments-additional-payment-methods',
+					'parent' => 'woocommerce-settings-payments',
+					'title'  => __( 'Add new payment methods', 'woocommerce-payments' ),
+					'path'   => '/payments/additional-payment-methods',
```

Comment:
There's probably a more convenient path for this screen. I've added this one in the meantime but glad to update it.